### PR TITLE
zrythm: 1.0.0-alpha.28.1.3 -> 1.0.0-beta.3.10.1

### PR DESCRIPTION
**Pull request by fogti**
###### Description of changes

https://github.com/zrythm/zrythm/compare/v1.0.0-alpha.28.1.3...v1.0.0-beta.3.9.1

This should now work as-is; it targets the staging branch not due to massive rebuilds, but because `sord` was merged into there.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### Blockers

* #182618 for libadwaita and libpanel
* #194796 for sord
* ~~#190724 for zix~~

### CODE REVIEW

File: pkgs/applications/audio/zrythm/default.nix [hunk 1]
```diff
@@ -84,17 +89,19 @@ stdenv.mkDerivation rec {
     python3.pkgs.sphinx
     texi2html
     wrapGAppsHook
-    cmake
+    cmake # put cmake at the end to make sure meson is used
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/zrythm/default.nix:92]: We should rather use ``dontUseCmakeConfigure``

**Comment by SuperSandro2000** [comment]
The build is failing. Can you take a look?

**Comment by zseri** [comment]
Yeah I've already seen it, probably incompatible with the currently-in-staging `libpanel` version. I might need to bump it again.

**Comment by zseri** [comment]
https://logs.nix.ci/?key=nixos/nixpkgs.190730&attempt_id=11e3b515-f7b3-44f4-9bde-3257c37ddb7f
```
Run-time dependency fftw3 found: YES 3.3.10
Run-time dependency fftw3_threads found: NO (tried pkgconfig and cmake)
Run-time dependency fftw3f_threads found: NO (tried pkgconfig and cmake)
meson.build:969:6: ERROR: C shared library 'fftw3_threads' not found
A full log can be found at /build/source/build/meson-logs/meson-log.txt
error: builder for '/nix/store/6kh0r20x9piyl46lhzd3b8rfdd9hh62r-zrythm-1.0.0-beta.3.9.1.drv' failed with exit code 1;
```

the manually hardwired fftw3 turned into a fatal error, this must be fixed properly, but idk how...
@magnetophon do you have any idea where this problem originally came from?

**Comment by alex-tee** [comment]
@zseri I added the following options

```meson
option (
  'fftw3_threads_separate',
  type: 'boolean',
  value: true,
  description: 'Whether to look for fftw3_threads pkgconfig files (or libs directly if not found) separate from the main fftw3 one.')

option (
  'fftw3_threads_separate_type',
  type: 'string',
  value: 'auto',
  description: 'Method to look for separate fftw3_threads. Either "library" or "dependency" or "auto".')

option (
  'fftw3f_separate',
  type: 'boolean',
  value: true,
  description: 'Whether to look for fftw3f separate of fftw3')
```

every distro packages fftw in various different ways so this makes my life easier than having to guess and do hacks in my meson.build. it's the packager's job now to specify these

zrythm just needs to call `fftw_make_planner_thread_safe()` and  `fftwf_make_planner_thread_safe()` but every distro packages fftw differently e.g. sometimes there are no pkgconfig files, sometimes the threads are part of the main pkgconfig and sometimes separate, etc.

**Comment by zseri** [comment]
@alex-tee thx

**Pull request closed without merging**